src/core/core.adapters.js

```diff
@@ -112,4 +120,6 @@ module.exports._date = {
 	_create: function(value) {
 		return value;
 	}
-};
+});
+
```

Comment:
I would add a static method to abstract how methods should be overridden, for example:

```javascript
DateAdapter.override = function(members) {
    helpers.extend(adapters._date.prototype, members);
} 
```

So external (and our moment) adapters would simply call:

```javascript
Chart._adapters._date.override({
    parse: function(...) {
        // ...
    }
});
```